canvases/dark-repl/main.dark

```diff
@@ -1,5 +1,6 @@
 [<HttpHandler("GET", "/")>]
-let _handler _req = Http.redirectTo "/assets/index.html"
+let _handler _req =
+  PACKAGE.Darklang.Stdlib.Http.redirectTo "/assets/index.html"
```

Comment:
The redirect isn't working currently, I get `Header must be a string`error 
but when I try to access `http://dark-repl.dlio.localhost:11003/assets/index.html`it works fine

Reply:
Hmm this may be resolved if you rebase. I think that [this](#4923) PR would likely fix the issue.

Reply:
specifically this line: https://github.com/OceanOak/dark/compare/spike-4...darklang:dark:main#diff-2e69a7ca467f30e522c6179c7bb8f6b3eb6898577e88a197aa9fd596a9207a0eR84

(in other words: the function broke when we ported it to .dark package, and that PR fixed it)

Reply:
Thanks!